tests/topology/testHalfEdgeDataStructure.cpp

```diff
@@ -47,6 +47,9 @@ typedef HalfEdgeDataStructure::PolygonalFace    PolygonalFace;
 typedef HalfEdgeDataStructure::Edge             Edge;
 typedef HalfEdgeDataStructure::Arc              ArcT; //Arc already defined in wingdi.h
 typedef HalfEdgeDataStructure::VertexIndexRange VertexIndexRange;
+typedef HalfEdgeDataStructure::Size Size;
```

Comment:
I would suggest `using Foo::Bar` rather than `typedef Foo::Bar Bar` to let the ADL doing the job

Reply:
Is ADL not only related to function name lookup ?